script/update.sh

```diff
@@ -38,4 +38,3 @@ npm test
 git add index.json
 git commit -am "feat: update history (electron@$ELECTRON_SHA)"
```

Comment:
This not redirects to updated electron sha 🙈

Reply:
Ugh. Thanks GitHub. Do you think we should keep this, or remove it?

Reply:
flip, 50/50, not sure, not big use case for me, it's can be found in the inside commit

Reply:
Okay, I'll leave it for now since this is ready and we can change our minds later